src/ipa_pc/data_structures.rs

```diff
@@ -105,13 +106,16 @@ impl<G: AffineCurve> PCCommitment for Commitment<G> {
     }
 
     fn size_in_bytes(&self) -> usize {
-        algebra_core::to_bytes![G::zero()].unwrap().len() / 2
+        to_bytes![G::zero()].unwrap().len() / 2
```

Comment:
I am advocating for the CanonicalSerialize trait, because for example in this case we'd be able to just use `G::SERIALIZED_SIZE` (or `G::UNCOMPRESSED_SIZE` if you'd want an uncompressed size). 

From a performance perspective, this call right now does multiple allocations (get zero element + convert to bytes) vs getting the trait's constant